### PR TITLE
Fix midair collision

### PR DESCRIPTION
**Pull request by doug-wade**
I broke the build :cry:.  This fixes it.  Thanks to @gigabo for [catching it](https://github.com/lerna/lerna/pull/599#issuecomment-280487962)

### CODE REVIEW

File: test/PublishCommand.js [hunk 1]
```diff
@@ -729,27 +729,27 @@ describe("PublishCommand", () => {
         ]],
         [ChildProcessUtilities, "exec", { nodeCallback: true }, [
           { args: ["cd " + escapeArgs(path.join(testDir, "packages/package-1")) + " && npm publish --tag lerna-temp"] },
-          { args: ["cd " + escapeArgs(path.join(testDir, "packages/package-2")) + " && npm publish --tag lerna-temp", {env: {"npm_config_registry":"https://my-private-registry"}}] },
           { args: ["cd " + escapeArgs(path.join(testDir, "packages/package-3")) + " && npm publish --tag lerna-temp", {env: {"npm_config_registry":"https://my-private-registry"}}] },
-          { args: ["cd " + escapeArgs(path.join(testDir, "packages/package-4")) + " && npm publish --tag lerna-temp", {env: {"npm_config_registry":"https://my-private-registry"}}] }
+          { args: ["cd " + escapeArgs(path.join(testDir, "packages/package-4")) + " && npm publish --tag lerna-temp", {env: {"npm_config_registry":"https://my-private-registry"}}] },
+          { args: ["cd " + escapeArgs(path.join(testDir, "packages/package-2")) + " && npm publish --tag lerna-temp", {env: {"npm_config_registry":"https://my-private-registry"}}] }
```
Comment (gigabo) [inline test/PublishCommand.js]: :squint: `[1,3,4,2]`?

Is that a consistent order, or are these just all racing against each other?

Comment (gigabo) [inline test/PublishCommand.js]: Whelp, the build passed, so... fixed forever! 😁

Comment (doug-wade) [inline test/PublishCommand.js]: I'm p sure its a consistent order.  2 and 4 have a dependency on 1, and 1 and 3 have no dependencies, so there are two batches: `[[1,3], [2,4]]`.  As to whether 1 always goes before 3 and 4 always goes before 2, I'm less certain, but in my ten or so runs they are consistent, and all the ci suites passed, so it seems so?

Comment (gigabo) [inline test/PublishCommand.js]: That makes sense.  Thanks for checking!

**Pull request merged**